packages/terra-form-select/src/_Frame.jsx

```diff
@@ -196,6 +201,8 @@ class Frame extends React.Component {
       onFocus: this.handleInputFocus,
       onBlur: this.handleInputBlur,
       onMouseDown: this.handleInputMouseDown,
+      required,
+      'aria-required': required,
```

Comment:
Does this do anything strange for `multiple` and `tag` variants? Options can be selected and the input will be empty but marked as required.

Reply:
hmm, I hadn't thought about that scenario. I'll have to check it out.

Reply:
Updated: https://github.com/cerner/terra-core/pull/2398/commits/a8f8165848a4b4bcbd0f39ef434c986c46244490

If you select atleast 1 option in the multiple or tag variants, I remove the required attributes from the input since you've essentially entered a value.